iterative/aws/provider.go

```diff
@@ -44,7 +48,7 @@ func ResourceMachineCreate(ctx context.Context, d *schema.ResourceData, m interf
 	if ami == "" {
 		ami = "iterative-cml"
 	}
-
+	fmt.Println(region)
```

Comment:
skipped by accident?

Reply:
Yes, It will be gone by the "ready for review" time (early comments are always welcome). Also, it didn't even work 😄

Reply:
I know! same happened to me long time ago